Gruntfile.js

```diff
@@ -136,21 +137,14 @@ module.exports = function(grunt) {
 
 
     tests: {
-      jqlite: 'karma-jqlite.conf.js',
-      jquery: 'karma-jquery.conf.js',
+      'jqlite': 'karma-jqlite.conf.js',
+      'jquery': 'karma-jquery.conf.js',
```

Comment:
These 2 lines don't need changes, do they? The same with `docs`. Let's only add quotes if needed, that reduces Git churn.